src/engine/enginedeck.cpp

```diff
@@ -30,23 +30,15 @@ EngineDeck::EngineDeck(const ChannelHandleAndGroup& handle_group,
                        UserSettingsPointer pConfig,
                        EngineMaster* pMixingEngine,
                        EffectsManager* pEffectsManager,
-                       EngineChannel::ChannelOrientation defaultOrientation)
-        : EngineChannel(handle_group, defaultOrientation),
+                       EngineChannel::ChannelOrientation defaultOrientation,
+                       bool effectable)
+        : EngineChannel(handle_group, defaultOrientation,
+                        pEffectsManager, false, effectable),
           m_pConfig(pConfig),
-          m_pEngineEffectsManager(pEffectsManager ? pEffectsManager->getEngineEffectsManager() : NULL),
-          m_pInputConfigured(new ControlObject(ConfigKey(getGroup(), "input_configured"))),
```

Comment:
This must not be removed. It crashes now with vinyl control.

Reply:
whoops